libsawtooth/Cargo.toml

```diff
@@ -54,8 +55,11 @@ lmdb-store = ["lmdb-zero", "stores"]
 receipt-store = ["stores"]
 redis-store = ["redis", "stores"]
 stores = []
-validator-internals = ["hex", "openssl", "lmdb-zero"]
+validator-internals = ["cbor-codec","hex", "openssl", "lmdb-zero"]
```

Comment:
Missing a space here (also, lmdb-zero isn't in its alphabetical spot).

Reply:
fixed